pkg/controllers/networkinfo/networkinfo_controller.go

```diff
@@ -56,7 +56,7 @@ var (
 	nsMsgVPCNetCfgGetError        = newNsUnreadyMessage("Error happened to get VPC network configuration: %v", NSReasonVPCNetConfigNotReady)
 	nsMsgSystemVPCNetCfgNotFound  = newNsUnreadyMessage("Error happened to get system VPC network configuration: %v", NSReasonVPCNetConfigNotReady)
 	nsMsgVPCGwConnectionGetError  = newNsUnreadyMessage("Error happened to validate system VPC gateway connection readiness: %v", NSReasonVPCNetConfigNotReady)
-	nsMsgVPCGwConnectionNotReady  = newNsUnreadyMessage("System VPC gateway connection is not ready", NSReasonVPCNetConfigNotReady)
+	nsMsgVPCConnectionNotReady    = newNsUnreadyMessage("Neither of System VPC gateway connection nor service cluster is ready", NSReasonVPCNetConfigNotReady)
```

Comment:
Neither VPC system gateway nor service cluster is ready

Reply:
Updated with putting system before VPC, thanks. Agree the system is to describe the vpc instead of gateway.